cmd/lncli/commands.go

```diff
@@ -1650,6 +1612,56 @@ mnemonicCheck:
 	return nil
 }
 
+// capturePassword returns a password value that has been
```

Comment:
Excellent description! The line wrapping on the godoc looks like it breaks before 80 chars, but not a blocker

Reply:
Thanks!
I was wrapping at 72 because I remembered seeing it somewhere in the contributions guide. but that was for commit messages 🤦‍♀